Legal (#57)
userstyles-world/userstyles.world: privacy_policy.md

# GDPR PRIVACY POLICY OF USERSTYLES.WORLD
Last updated May 5, 2021

## INTRODUCTION

> Thank you for choosing to be part of our community at userstyles world (“company”, “we”, “us”, or “our”). 

Author: Oh we are not? :)

    15. HOW CAN YOU CONTACT US ABOUT THIS POLICY?  

## WHAT INFORMATION DO WE COLLECT? 

### **Personal information you disclose to us**

_In Short: We collect personal information that you provide to us such as email-address._  

We collect personal information that you voluntarily provide to us when registering at the Site, 
and further on by editing your profile at the Site.
The personal information that we collect depends on the context of your interactions 
with us and the Sites, the choices you make and the products and features you use. The personal information we collect can include the following: 
Name and Contact Data. We collect username, email address and social media username.
Credentials. We collect passwords. All of the Credentials are hashed and original values not stored.
All personal information that you provide to us must be true, complete and accurate, and you must 
notify us of any changes to such personal information.

### **Information automatically collected**  

_In Short: Some information – such as IP address and/or browser and device characteristics – is collected automatically when you visit our websites._  

> We automatically collect certain information when you visit, use or navigate the Site. This information does not reveal your specific identity (like your name or contact information) but may include device and usage information, such as your IP address, browser and device characteristics, operating system, language preferences, referring URLs, device name, country, location, information about how and when you use our Site and other technical information. This information is primarily needed to maintain the security and operation of our Sites, and for our internal analytic and reporting purposes. 

Author: NGINX Logs are included, and we need to adress the logs differently and make an statement it's flushed within another time period.

We use the information we collect or receive:  

    • To facilitate account creation and logon process [with your Consent]: If you choose to link your account with us to a third party account *(such as your Github or Codeberg account), we use the information you allowed us to collect from those third parties to facilitate account creation and logon process. See the section below headed "Social Logins" for further information. 

    • To send administrative information to you [for Service Purposes, Legal Reasons and/or possibly Contractual]: We may use your personal information to send you product, service and new feature information and/or information about changes to our terms, conditions, and policies.

    • Request Feedback [for our Service Purposes and/or with your Consent]: We may use your information to request feedback and to contact you about your use of our Sites 

    • To protect our Sites [for Service Purposes and/or Legal Reasons]: We may use your information as part of our efforts to keep our Sites safe and secure (for example, for fraud monitoring and prevention).

>     • To enable user-to-user communications [with your consent]: We may use your information in order to enable user-to-user communications with each user's consent.

Author: Good catch, got a little bit stuck with this whole rating system which could be seen as user-to-user communications, as it's direct feedback to another user(style author)

    • Vital Interests and Legal Rights. We may disclose your information where we believe it is necessary to investigate, prevent, or take action regarding potential violations of our policies, suspected fraud, situations involving potential threats to the safety of any person and illegal activities, or as evidence in litigation in which we are involved. 

    • With your Consent. We may disclose your personal information for any other purpose with your consent.

    • Other Users. When you share personal information (for example, by posting comments, contributions or other content to the Site) or otherwise interact with public areas of the Site. such personal information may be viewed by all users and may be publicly distributed outside the Site in perpetuity. Similarly, other users will be able to view descriptions of your activity, communicate with you within our Site, and view your profile.

## DO WE USE COOKIES AND OTHER TRACKING TECHNOLOGIES? 

> _In Short: No_

Reviewer: ```suggestion
_In Short: No._
```

> We don't use cookies for tracking technologies, however we uses the cookie to ensure
> authorization between the User and our Site.

Reviewer: ```suggestion
We don't use cookies for tracking technologies, however we use cookies to ensure
authorization between the User and our Site, only when the user is logged in.
```

## HOW DO WE HANDLE YOUR SOCIAL LOGINS?

_In Short: If you choose to register or log in to our websites using a social media account, we may have access to certain information about you._

> Our Sites offers you the ability to register and login using your third party social media account details (like your Github or Codeberg logins). Where you choose to do this, we will receive certain profile information about you from your social media provider. The profile Information we receive may vary depending on the social media provider concerned, but will often include your name, e-mail address, follower list, profile picture as well as other information you choose to make public.

Reviewer: ```suggestion
Our Site offers you the ability to register and login using your third party social media account details (like your Github, Gitlab, or Codeberg logins). When you choose to do this, we will receive certain profile information about you from your social media provider. The profile Information we receive may vary depending on the social media provider concerned, but will often include your name, email address, follower list, profile picture as well as other information you choose to make public, of which only username and email address will be used on the Site.
```

We will use the information we receive only for the purposes that are described in this privacy policy or that are otherwise made clear to you on the Site. Please note that we do not control, and are not responsible for, other uses of your personal information by your third party social media provider. We recommend that you review their privacy policy to understand how they collect, use and share your personal information, and how you can set your privacy preferences on their sites and apps.

## IS YOUR INFORMATION TRANSFERRED INTERNATIONALLY?

> _In Short: No, not to our knowingly._

Reviewer: ```suggestion
_In Short: No, not to our knowledge._
```

> Our server is provided by vultr.com, we recommend that you review their privacy policy to understand how they collect, use and share your personal information. Our server is located in Amsterdam, NL. All traffic from userstyles.world are processed by this server.

Reviewer: ```suggestion
Our server is provided by vultr.com, we recommend that you review their privacy policy to understand how they collect, use and share your personal information. Our server is located in Amsterdam, Netherlands. All traffic from userstyles.world is processed by this server.
```

> If you arem’t a resident in the European Economic Area, then these countries may not have data protection or other laws as comprehensive as those in your country. We will however take all necessary measures to protect your personal information in accordance with this privacy policy and applicable law. 

Reviewer: ```suggestion
If you aren’t a resident in the European Economic Area, then these countries may not have data protection or other laws as comprehensive as those in your country. We will however take all necessary measures to protect your personal information in accordance with this privacy policy and applicable law. 
```

## WHAT IS OUR STANCE ON THIRD-PARTY WEBSITES?

> _In Short: We are not responsible for the safety of any information that you share with third-party providers who advertise, but are not affiliated with, our websites._

Reviewer: ```suggestion
_In Short: We are not responsible for the safety of any information that you share with third-party providers who advertise, but are not affiliated with, our website._
```

> The Sites may contain advertisements from third parties that are not affiliated with us and which may link to other websites, online services or mobile applications. We cannot guarantee the safety and privacy of data you provide to any third parties. Any data collected by third parties is not covered by this privacy policy. We are not responsible for the content or privacy and security practices and policies of any third parties, including other websites, services or applications that may be linked to or from the Sites. You should review the policies of such third parties and contact them directly to respond to your questions.

Reviewer: ```suggestion
The Site may contain advertisements from third parties that are not affiliated with us and which may link to other websites, online services, or mobile applications. We cannot guarantee the safety and privacy of data you provide to any third parties. Any data collected by third parties is not covered by this privacy policy. We are not responsible for the content or privacy and security practices and policies of any third parties, including other websites, services, or applications that may be linked to or from the Site. You should review the policies of such third parties and contact them directly to respond to your questions.
```

## HOW LONG DO WE KEEP YOUR INFORMATION? 

_In Short: We keep your information for as long as necessary to fulfill the purposes outlined in this privacy policy unless otherwise required by law._

We will only keep your personal information for as long as it is necessary for the purposes set out in this privacy policy, unless a longer retention period is required or permitted by law (such as tax, accounting or other legal requirements). No purpose in this policy will require us keeping your personal information for longer than the period of time in which you have an account with us.

When we have no ongoing legitimate business need to process your personal information, we will delete it, or, if this is not possible (for example, because your personal information has been stored in backup archives), then we will securely store your personal information and isolate it from any further processing until deletion is possible.

## HOW DO WE KEEP YOUR INFORMATION SAFE? 

_In Short: We aim to protect your personal information through a system of organizational and technical security measures._

> We have implemented appropriate technical and organizational security measures designed to protect the security of any personal information we process. However, please also remember that we cannot guarantee that the internet itself is 100% secure. Although we will do our best to protect your personal information, transmission of personal information to and from our Sites is at your own risk. You should only access the services within a secure environment. 

Reviewer: ```suggestion
We have implemented appropriate technical and organizational security measures designed to protect the security of any personal information we process. However, please also remember that we cannot guarantee that the internet itself is 100% secure. Although we will do our best to protect your personal information, transmission of personal information to and from our Site is at your own risk. You should only access the service within a secure environment. 
```
